### PR TITLE
[WIP]✨[hk] buildings

### PR DESCRIPTION
**Pull request by Cr4zyc4k3**
**What kind of update does this PR provide?** *Please check*

- [ ] new translations / updated translations / translation fixes
- [X] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**What is included in your update?**
* Hotkeys for buildings

### CODE REVIEW

**Comment by deoxis9001** [comment]
```
        "building": {
            "goto": {
                "dispatchCenter": "Centre de Traitement des Appels",
                "nextBuilding": "Bâtiment suivant",
                "previousBuilding": "Bâtiment précedent",
                "title": "aller à"
            },
            "title": "Bâtiments"
        },
```
        fr_FR

File: src/modules/hotkeys/i18n/en_AU.root.json [hunk 1]
```diff
@@ -35,6 +35,15 @@
             },
             "title": "General"
         },
+        "building": {
```
Comment (itayo) [inline src/modules/hotkeys/i18n/en_AU.root.json:38]: `       "building": {
            "goto": {
                "dispatchCenter": "larmcentral",
                "nextBuilding": "Nästa byggnad",
                "previousBuilding": "Föregående byggnad",
                "title": "gå till"
            },
            "title": "Byggnad"
        },`

sv_SE

Comment (Cr4zyc4k3) [reply src/modules/hotkeys/i18n/en_AU.root.json:38]: Hope you meant se_SE and not en_AU :smile:

**Comment by itayo** [comment]
Oopa my bad yes it was ment for sv_SE

Den fre 16 sep. 2022 12:23Mike ***@***.***> skrev:

File: src/modules/hotkeys/i18n/en_AU.root.json (hunk 1 above)
Comment (itayo) [reply src/modules/hotkeys/i18n/en_AU.root.json:38]: yup oopsie :)

**Comment by itayo** [comment]
I have reviewed commit bb00708 regarding sv_SE and that looks fine to me

File: src/modules/hotkeys/i18n/en_AU.root.json (hunk 1 above)
Comment (itayo) [reply src/modules/hotkeys/i18n/en_AU.root.json:38]: OH wait there is a typoo by my side i see now

Comment (itayo) [reply src/modules/hotkeys/i18n/en_AU.root.json:38]: "dispatchCenter": "Larmcentral"

**Pull request closed without merging**